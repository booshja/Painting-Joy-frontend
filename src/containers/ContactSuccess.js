// dependencies
import React from "react";
import styled from "styled-components";
// components
import { GoBack, PageTitle } from "../components";
// assets
import MailSuccess from "../assets/jess-moe-PbUzvmAmXmk-unsplash.jpg";

const StyledContactSuccess = styled.div`
    display: flex;
    flex-direction: column;
    align-items: center;
`;

const StyledMessage = styled.p`
    font-family: "News Cycle", sans-serif;
    width: 90%;
    font-size: 1.4rem;
    letter-spacing: 1px;
    text-align: center;
`;

const StyledImg = styled.img`
    width: 80%;
`;

const ContactSuccess = () => {
    return (
        <StyledContactSuccess>
            <PageTitle>Contact Me</PageTitle>
            <StyledMessage>
                Thank you for your message! I've got it saved and I'll reply
                just as soon as I get all this paint off my hands!
            </StyledMessage>
            <GoBack to={"/"} />
            <StyledImg
                src={MailSuccess}
                alt="Envelope with dried flowers on a table"
            />
        </StyledContactSuccess>
    );
};

export default ContactSuccess;
