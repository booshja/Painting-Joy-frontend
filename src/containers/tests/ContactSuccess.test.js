import { render } from "@testing-library/react";
import ContactSuccess from "../ContactSuccess";

/** Smoke Test */
it("renders without crashing", () => {
    render(<ContactSuccess />);
});

/** Snapshot Test */
it("matches snapshot", () => {
    const { asFragment } = render(<ContactSuccess />);
    expect(asFragment()).toMatchSnapshot();
});
