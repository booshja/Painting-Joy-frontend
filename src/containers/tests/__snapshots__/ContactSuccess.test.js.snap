// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="sc-eWnToP fcnWwj"
  >
    <h1
      class="sc-fWWYYk cwzJaj"
    >
      Contact Me
    </h1>
    <p
      class="sc-kTCsyW igGrGt"
    >
      Thank you for your message! I've got it saved and I'll reply just as soon as I get all this paint off my hands!
    </p>
    <button
      class="sc-fXgAZx fGPCkt"
    >
      <i
        class="fas fa-long-arrow-alt-left"
      />
      Back
    </button>
    <img
      alt="Envelope with dried flowers on a table"
      class="sc-dvUynV lleqec"
      src="jess-moe-PbUzvmAmXmk-unsplash.jpg"
    />
  </div>
</DocumentFragment>
`;
